ui/src/index.css

```diff
@@ -11,3 +11,8 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
+.progress-bar-content {
```

Comment:
I removed this and it still works in my tests... Can you confirm, please?

Reply:
yes, It works on all the major font size options of browsers, but when the user goes too crazy and sets his default font size to 30px or more it breaks the layout.

what do you suggest? is it ok to add or I can remove it.

Reply:
Can't we add it to the scope of the player, instead of globally? We could add it to https://github.com/navidrome/navidrome/blob/0063720cc2636dbca55b66902764b136d7b6b622/ui/src/audioplayer/Player.js#L42:

```js
    player: {
      display: (props) => (props.visible ? 'block' : 'none'),
      '$.progress-bar-content': {
....
      }
    },
```

Reply:
I think there is no way to select the `.progress-bar-content` from our side, we can only change this behavior by submitting the  PR to [react-jinkie-player](https://github.com/lijinke666/react-music-player) repo or by applying the global style.

Reply:
The syntax in my previous comment was wrong. I just did in the latest commit in master: https://github.com/navidrome/navidrome/commit/9a509c749a974b7b25e1f1d4e5c09feb92853003

Reply:
yeah, I see it is working I will submit the new commit now :)

Reply:
Also, please fix the merge conflict.